configuration/concepts/operative_progress.csv

```diff
@@ -0,0 +1,22 @@
+Uuid,Void/Retire,Same as mappings,Fully specified name:en,Short name:en,Fully specified name:es,Short name:es,Description:en,Description:es,Data class,Data type,Complex data handler,Answers,Members,Allow decimals,Units,Normal low,Normal high,Absolute low,Absolute high,_version:1,_order:1100
+a957ff9c-80f1-4f31-8937-3bef0cfb8c0e,,,Pre-Operative Diagnosis,Pre-Op Diagnosis,Diagnóstico preoperatorio,Diagnóstico preoperatorio,,,Misc,Text,,,,,,,,,,,
+f348e4b5-4124-4169-966a-2dff2a944566,,,post-Operative Diagnosis,post-Operative Diagnosis,Diagnóstico postoperatorio,Diagnóstico postoperatorio,,,Misc,Text,,,,,,,,,,,
+27fe6714-0bc6-4435-adb0-818538abe42c,,,Procedure,Procedure,Procedimiento,Procedimiento,,,Misc,Text,,,,,,,,,,,
+acda413c-56cc-48ab-825d-af58cb947f15,,,Assistants,Assistants,Asistentes,Asistentes,,,Misc,Text,,,,,,,,,,,
+3ca389b4-6b9d-487c-bdd9-73d2cfeeab0c,,,General,General,General,General,,,Misc,N/A,,,,,,,,,,,
+5392d074-eed3-4f88-a8e9-5bae333e7589,,,Monitored Anesthesia Care,Monitored Anesthesia Care,Atención de anestesia supervisada,Atención de anestesia supervisada,,,Misc,N/A,,,,,,,,,,,
+84ba7929-3714-4023-843a-2b1814563207,,,Knee Block,Knee Block,bloque de rodilla,bloque de rodilla,,,Misc,N/A,,,,,,,,,,,
+7dd25939-b4e5-4657-8c69-ac4993f183b5,,,Axillary Block,Axillary Block,Bloque Axilar,Bloque Axilar,,,Misc,N/A,,,,,,,,,,,
+0f087406-b092-419c-a9ae-bd32c9e29a43,,,Interscalene Block,Interscalene Block,Bloque interescalénico,Bloque interescalénico,,,Misc,N/A,,,,,,,,,,,
+20fafddb-bfd4-421e-acfb-639454741e94,,,Intravenous Block,IV Block,Bloque intravenoso,bloque CI,,,Misc,N/A,,,,,,,,,,,
+3dbb2d82-ee96-43a5-a905-0401c2d8b1a9,,,Local anesthesia and Sedation,Local anesthesia and Sedation,Anestesia local y Sedación,Anestesia local y Sedación,,,Misc,N/A,,,,,,,,,,,
+ea6ac7de-d7cf-4105-aadb-b1e489536a7e,,,Local anesthesia,Local anesthesia,Anestesia local,Anestesia local,,,Misc,N/A,,,,,,,,,,,
+3017f680-5c3c-4f7b-94e1-6afda9e07eaf,,,Anesthesia Type,Anesthesia Type,Tipo de anestesia,Tipo de anestesia,,,ConvSet,N/A,,,General; Monitored Anesthesia Care; Knee Block; Axillary Block; Interscalene Block; Intravenous Block; Local anesthesia and Sedation,,,,,,,,
+b5ab7bc0-bffb-4dc8-b832-2e1713f06944,,,Estimated Blood Loss,EBL,Pérdida de sangre estimada,LBE,,,Misc,Numeric,,,,,mls,,,,,,
+64fe56e6-5b52-4cdb-adc2-b6d50bfdee63,,,Complications,Complications,Complicaciones,Complicaciones,,,Misc,Text,,,,,,,,,,,
+08f1f638-b3c7-4040-8ae9-c95a5df59755,,,Specimens,Spcmns,especímenes,Spcmns,,,Misc,Text,,,,,,,,,,,
+59185273-c5b8-4e73-8ba2-c68710f6c507,,,Drains,Drains,Drenajes,Drenajes,,,Misc,Text,,,,,,,,,,,
+a1da87f9-c4af-4598-accd-489268a2d325,,,Condition upon Discharge from OR,Condition upon Discharge,Condición al alta del quirófano,Condición al alta,,,Misc,Text,,,,,,,,,,,
```

Comment:
Change to 'Condition upon discharge from OR'
And let's use the same for Short name

Reply:
Done